pkg/lib/config/authentication_flow_captcha.go

```diff
@@ -51,7 +51,11 @@ type AuthenticationFlowCaptchaProvider struct {
 }
 type AuthenticationFlowCaptcha struct {
 	Mode     AuthenticationFlowCaptchaMode      `json:"mode,omitempty"`
-	Provider *AuthenticationFlowCaptchaProvider `json:"provider,omitempty"`
+	Provider *AuthenticationFlowCaptchaProvider `json:"provider,omitempty" nullable:"true"`
```

Comment:
Remove the struct tag `nullable:"true"` here. Instead, we should add `nullable:"true"` to the structs that contain `AuthenticationFlowCaptcha`